2_Workspace/wishock/state.cpp

```diff
@@ -226,8 +226,8 @@ void Wifi_Connect (void)
   gLedFlag = LED_STATUS_BLINK;
   WiFi.begin();
   #ifdef DEBUG
-  WiFi.printDiag(Serial);
-  Serial.print("\r\nWiFi connecting");
+    WiFi.printDiag(Serial);
+    Serial.print("\r\nWiFi connecting");
```

Comment:
Chỗ này là sao nhở?

Reply:
cái này là in ra màn hình cái ssid với pass nhận được từ smart config

Reply:
Ý là hỏi tại sao chêm 2 space vào á :))